sdk/storage/Azure.Storage.Files.Shares/api/Azure.Storage.Files.Shares.netstandard2.0.cs

```diff
@@ -720,6 +726,16 @@ public enum ShareTraits
         None = 0,
         Metadata = 1,
     }
+    public partial class SMB
+    {
+        public SMB() { }
+        public Azure.Storage.Files.Shares.Models.SmbMultichannel Multichannel { get { throw null; } set { } }
+    }
+    public partial class SmbMultichannel
+    {
+        public SmbMultichannel() { }
+        public bool? Enabled { get { throw null; } set { } }
+    }
```

Comment:
This is quite a lot of wrapping around single boolean. 
For the SMB->SmbMultichannel. How about we do SMBSettings/SmbSettings? { bool MultichannelEnabled; }?
For the ProtocolSettings -> SMB - are we going to have more protocols in the future ?

Btw. these all models seem to be coming from generator. Do we expose them or are we going to write some models by hand?

Reply:
The xml structure is:
```
    <ProtocolSettings> 
        <SMB> 
            <Multichannel> 
                <Enabled>true|false</Enabled> 
            </Multichannel> 
        </SMB> 
    </ProtocolSettings> 
```

I think we need to leave both the `SMB` and `Multichannel` layers.  If we don't, we'll have a real problem if they add sibling properties in the future.